package.json

```diff
@@ -69,5 +69,14 @@
   "resolutions": {
     "typescript": "^5.3.3"
   },
-  "version": "0.9.6-6"
+  "version": "0.9.6-6",
+  "dependencies": {
+    "doken-precomputer": "^0.0.4",
```

Comment:
Adding this might cause circular dependency as doken-precomputer also has this cord.js/types dependency.